### PR TITLE
Release v0.12.5

### PR DESCRIPTION
**Pull request by sphuber**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -181,9 +186,9 @@
 - REST API: update endpoints for structure visualization, calculation (includes retrieved input & output list), add endpoints for `UpfData` and more [[#977]](https://github.com/aiidateam/aiida_core/pull/977) [[#991]](https://github.com/aiidateam/aiida_core/pull/991)
 - Tests using daemon run faster [[#870]](https://github.com/aiidateam/aiida_core/pull/870)
 - Documentation: updated outdated workflow examples [[#948]](https://github.com/aiidateam/aiida_core/pull/948)
-- Documentation: updated import/export [[#994]](https://github.com/aiidateam/aiida_core/pull/994), 
-- Documentation: plugin quickstart [[#996]](https://github.com/aiidateam/aiida_core/pull/996), 
-- Documentation: parser example [[#1003]](https://github.com/aiidateam/aiida_core/pull/1003) 
+- Documentation: updated import/export [[#994]](https://github.com/aiidateam/aiida_core/pull/994),
```
Comment (ltalirz) [inline CHANGELOG.md:189]:
```suggestion
- Documentation: updated import/export [[#994]](https://github.com/aiidateam/aiida_core/pull/994)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -181,9 +186,9 @@
...
+- Documentation: plugin quickstart [[#996]](https://github.com/aiidateam/aiida_core/pull/996),
```
Comment (ltalirz) [inline CHANGELOG.md:190]:
```suggestion
- Documentation: plugin quickstart [[#996]](https://github.com/aiidateam/aiida_core/pull/996)
```

**Comment by ltalirz** [comment]
Just two commas to remove - let's see whether the tests still pass :-D

**Comment by sphuber** [comment]
And of course, it is `pymatgen`...

**Comment by ltalirz** [comment]
Anyhow, if we can fix the dependencies such that one can still install 0.12, this is probably worthwhile

**Comment by sphuber** [comment]
@ltalirz tests have passed now, except for the pre-commit one, but that is just a problem at the end with the Travis linter. I say we ignore this as it has nothing to do with the package itself

**Pull request merged**